File: FirstTask/JavaScript/Insertionsort/insertionsort.js
const array = [];
for(let i=0; i<10; i++){
    array.push(Math.floor(Math.random() * 100));
}
console.log("Array to sort: ", array);
let sortedarray = insertionsort(array);
console.log("Sorted Array: ", sortedarray)
function insertionsort(arr){
    for(let i=1; i<arr.length; i++){
        elem = arr[i];
        j = i-1;
        while(j>=0 && arr[j]>elem){
            arr[j+1]=arr[j];
            j--;
        }
        arr[j+1]=elem;
    }
    return arr;
}

File: FirstTask/Python/Insertionsort/Insertionsort.py
import random
def insertionsort(arr):
    for i in range(2, len(arr)):
        elem = arr[i]
        j = i - 1
        while j >= 0 and arr[j] > elem:
            arr[j + 1] = arr[j]
            j -= 1
        arr[j + 1] = elem
    return arr

def sortRandomArray():
    randomArray = [random.randint(1,100) for i in range(10)]
    print("Array to sort: ", randomArray)
    sortedArray = insertionsort(randomArray)
    print("Array after sorting: ",sortedArray)

sortRandomArray()